boards/BMS/Src/main.c

```diff
@@ -228,7 +232,7 @@ int main(void)
 
     Io_LTC6813_Init(&hspi2, SPI2_NSS_GPIO_Port, SPI2_NSS_Pin);
     App_AccumulatorVoltages_Init(Io_CellVoltages_GetRawCellVoltages);
-    cell_monitor = App_Accumulator_Create(
```

Comment:
Forgot to change the name `cell monitor` to `accumulator` in the last PR. Should be fixed now.